calcom/cal.com
# Recurring Events

## What does this PR do?

Recurring events FTW!

Implements #2180 

## Type of change


## How should this be tested?

- Go to event type and create a new one
- Go to the newly created event type configuration and open Advanced Setting
- Check the Recurring event option, you can leave the default options or change them as you desire.
- Book a meeting with the recurring event type just configured
- See all correspondent bookings created in the Bookings page
